packages/flutter_tools/lib/src/commands/run.dart

```diff
@@ -187,6 +187,15 @@ class RunCommand extends RunCommandBase {
         hide: true,
         help: 'Whether to automatically invoke webOnlyInitializePlatform.',
       )
+      ..addFlag('fast-start',
+        negatable: true,
+        defaultsTo: false,
+        hide: true,
+        help: 'Whether to quickly bootstrap applications with a minimal app. '
+              'Currently this is only support on Android devices run from the '
+              'command line. This option cannot be pared with --start-paused or '
```

Comment:
typo? paired?

Also - is this a temporary restriction? IDEs need to use `--start-paused` when debugging.

Reply:
Because of https://github.com/flutter/flutter/issues/45424 we can't yet hot restart an isolate into a paused state (or maybe we can, and it just need a different configuration?). Until that is resolved I can't support IDEs, but it should be fixable...